docs/ros/config/yaml.mdx

```diff
@@ -261,14 +266,23 @@ At this point, with just the **serial_number** and **system** defined: our robot
 
 Every robot platform has unique structures such as versatile sensor mounting solutions, wireless charging receivers, and waterproofing enclosures; we refer to these as attachments.
 
+#### Joystick Controller
+We support two types of controllers:
+  - **ps4**: standard Playstation4 controller.
+  - **logitech**: Logitech F710
+
+```yaml
+controller: ps4 # or logitech
+```
+
 #### Attachments
 
 There are three types of attachments:
```

Comment:
This says there are 3 types but the list below has 4 with two labelled as 2.  Should that be updated to be 4?